xDiffCollection/Classes/CollectionBin.swift

```diff
@@ -68,7 +73,11 @@ public struct CollectionBin<T, Backstorage: RangeReplaceableCollection>: Collect
     }
     
     private func appendingElement(_ element: T) -> CollectionBin {
-        return updating(with: _backstorage + [element])
+        var resp = _backstorage + [element]
```

Comment:
here is another thing.. I'm putting here. Because I can't insert the comment above but: 
there is a line above 
`return BinUpdateResult(bin: appendingElement(element), changes: CollectionChanges(addedIndexes: [endIndex]))` 

So let's see your linearInsert changes and insert at index 1.... but the number of elements 10... so the changes will return only the last index.. But technically its not right, do you agree? we have changed all the elements from the index at which the element has been placed.. What do you think?